compose/fastapi/entrypoint

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
```

Comment:
Why not just use docker compose's  `healthcheck` and `depends_on` properties?

Reply:
because the Redis gives you a signal that it is ready when the container is UP.
But the topic is that it is not really ready for doing connections.

For example, Postgres does.

That's why we need to check explicitly

Reply:
What do you think?